whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapExceptionMapper.java

```diff
@@ -68,36 +55,30 @@ public Response toResponse(final Exception exception) {
             return Response.status(HttpServletResponse.SC_BAD_REQUEST).entity(createErrorEntity(HttpServletResponse.SC_BAD_REQUEST, exception.getMessage())).build();
         }
 
-        if (exception instanceof EmptyResultDataAccessException) {
-            return Response.status(HttpServletResponse.SC_NOT_FOUND).entity(createErrorEntity(HttpServletResponse.SC_NOT_FOUND, "not found")).build();
-        }
-
-        if (exception instanceof QueryException) {
-            if ( ((QueryException) exception).getCompletionInfo() == QueryCompletionInfo.BLOCKED) {
-                return createErrorResponse(Response.Status.TOO_MANY_REQUESTS, exception.getMessage());
-            }
-            return Response.status(HttpServletResponse.SC_BAD_REQUEST).entity(createErrorEntity(HttpServletResponse.SC_BAD_REQUEST, exception.getMessage())).build();
-        }
 
         if (exception instanceof IllegalStateException) {
             return Response.status(HttpServletResponse.SC_INTERNAL_SERVER_ERROR).entity(createErrorEntity(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, exception.getMessage())).build();
```

Comment:
You can get rid of this as well, as its the SC_INTERNAL_SERVER_ERROR is the default behaviour